scripts/sync.py

```diff
@@ -161,7 +166,15 @@ def do(args, Lochness):
                 lochness.attempt(Module.sync, Lochness, subject, dry=args.dry)
 
     # annonymize PII
-    dpanonymize.lock_lochness(Lochness, pii_table_loc=Lochness['pii_table'])
```

Comment:
So will it no longer be triggered from lochness in future?

Reply:
Yeah... We have invested a good chuck of time on `dpanonymize`, but currently we are not removing any information from any datatypes using `dpanonymize`. (We are transferring selected datatype files under PROTECTED to AWS using `selective_sync` functionality)